numpy/core/tests/test_records.py

```diff
@@ -206,6 +206,15 @@ def test_pickle_2(self):
         assert_equal(a, pickle.loads(pickle.dumps(a)))
         assert_equal(a[0], pickle.loads(pickle.dumps(a[0])))
 
+    def test_objview_record(self):
+        # https://github.com/numpy/numpy/issues/2599
+        dt = np.dtype([('foo', 'i8'), ('bar', 'O8')])
```

Comment:
We have deprecated `'O8'`, better to use `'O'`, or this test will break pretty soon.